openmmforcefields/generators/template_generators.py

```diff
@@ -1322,7 +1322,11 @@ def INSTALLED_FORCEFIELDS(cls):
             # impropers and one with Amber-style impropers. The latter requires a special handler
             # (`AmberImproperTorsionHandler`) that is not shipped with the toolkit. See
             # https://github.com/openforcefield/amber-ff-porting/tree/0.0.3
-            if root.startswith("ff14sb") and 'off_impropers' not in root:
+            if (
+                (root.startswith("ff14sb") and 'off_impropers' not in root)
+                or root.startswith("tip")
+                or root.startswith("opc")
+            ):
```

Comment:
We recently added OPC, but as far as I'm aware there isn't much demand for anything not prefixed with this. OPC3, TIPnP-fb, and some other TIPnP are the only ones that has been discussed lately